addons/finger/stringtable.xml

```diff
@@ -58,6 +60,7 @@
         </Key>
         <Key ID="STR_ACE_finger_keyComb_description">
             <English>Points, and shows a virtual marker of where you are looking to nearby units. Can be held down.</English>
+            <German>Zeigt anderen, nahen Spielern die eigene Blickrichtung mittels eines Symbols. Kann dauerhaft ausgeführt werden.</German>
```

Comment:
`Zeigt in der Nähe befindlichen Spielern die eigene Blickrichtung mit Hilfe eines Kreissymbols an.`